Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1948,7 +1948,8 @@ def gen2018HiMix(fragment,howMuch):
                                       '--data' : '',
                                       '--scenario' : 'pp',
                                       '--eventcontent' : 'MINIAOD,DQM',
-                                      '--datatier' : 'MINIAOD,DQMIO'
+                                      '--datatier' : 'MINIAOD,DQMIO',
+                                      '--customise_unsch' : 'PhysicsTools/PatAlgos/slimming/customizeMiniAOD_HcalFixLegacy2016.customizeAll'
```

Comment:
@fabozzi @prebello 
please confirm that this modifier approach is OK for the workflow submission

Reply:
It looks like this is reused later in the 2017 setup
``` steps['REMINIAOD_data2017'] = merge([{'--era' : 'Run2_2017,run2_miniAOD_94XFall17'},steps['REMINIAOD_data2016']]) ```
That part should be modified to undo the change for --customise_unsch

Reply:
I was still working on 9_4_1 and I missed that. will fix